content/en/docs/reference/using-api/api-concepts.md

```diff
@@ -284,6 +284,36 @@ Content-Type: application/json
 <followed by regular watch stream starting from resourceVersion="10245">
 ```
 
+## Response compression
+
+{{< feature-state for_k8s_version="v1.16" state="beta" >}}
+
+`APIResponseCompression` is an option that allows the API server to compress the responses for **get**
+and **list** requests, reducing the network bandwidth and improving the performance of large-scale clusters.
+It is enabled by default since Kubernetes 1.16, but can be disabled by setting the `--feature-gates`
+flag on the API server to `APIResponseCompression=false`.
```

Comment:
```suggestion
It is enabled by default since Kubernetes 1.16 and it can be disabled by including
`APIResponseCompression=false` in the `--feature-gates` flag on the API server.
```